test/data/e2e/create/cluster/data/kfd.yaml

```diff
@@ -4,18 +4,18 @@
 
 version: v1.24.1
```

Comment:
should this be bumped to 1.25 and the modules, installers, etc. to the respective version?

Reply:
yes but I would leave that in a dedicated PR

Reply:
ok